pulsar-client-admin-api/src/main/java/org/apache/pulsar/policies/data/loadbalancer/ResourceUsage.java

```diff
@@ -27,17 +27,10 @@
 public class ResourceUsage {
     public final double usage;
     public final double limit;
-    @EqualsAndHashCode.Exclude
```

Comment:
Nit: remove the import line for this annotation.

Reply:
there's no import for `@EqualsAndHashCode.Exclude`

Reply:
Oh, right, sorry about that.